### ISSUE TITLE
Queue for img2img upscale chain

### ISSUE THREAD

**Issue by Stibo** [Kryptortio/SDAtom-WebUi-client-queue-ext#2]
Great extension! I actually don't know if the extension already allows this, but is it possible to create a queue for SD Upscale and every step in the queue takes the previously generated image to upscale it even further? For every step different settings but every step takes the previously generated image? I hope you get what I mean.

**Comment by Kryptortio** [comment]
I don't think this is a common use case but you can do this with the [extension function](https://github.com/Kryptortio/SDAtom-WebUi-us#script-extensions). If you add this:

```
let loadJsonOld = loadJson;
loadJson = async function (p_json) {
	if(document.querySelector('#img2img_gallery img')) { // If there is an image generated
		document.querySelector('#image_buttons_img2img #img2img_tab').click(); // Click send to i2i from i2i button
		await sleep(2000); // Wait 2 seconds
	}
	loadJsonOld(p_json);
}
```
In the extension box and reload the page, every time settings are loaded it will first click the send to i2i button (if there is an image generated already). When you are done just remove the extension again and reload the page to disable the it.
